cli/src/print.rs

```diff
@@ -37,14 +37,16 @@ pub fn print_update_message() {
     eprintln!("{:-^50}\n\n", "");
 }
 
-pub fn print_sc_help(app: &mut Command, subcommand: &str) {
-    for sc in app.get_subcommands_mut() {
-        if sc.get_name() == subcommand {
-            let _ = sc.print_help();
-            break;
+pub fn print_sc_help(mut app: &mut Command, subcommands: &[&str]) {
+    for subcommand in subcommands {
+        match app.get_subcommands_mut().find(|sc| &sc.get_name() == subcommand) {
```

Comment:
Is there a reason not to use the [`find_subcommand_mut`](https://docs.rs/clap/latest/clap/struct.App.html#method.find_subcommand_mut) method here? It is shorter and it also works with aliases.

```suggestion
        match app.find_subcommand_mut(subcommand) {
```

Reply:
I just used the same method that was used before without looking at the available API, find sounds good to me.